### DISCUSSION TITLE
绳网崩了？

### DISCUSSION THREAD

**Discussion by Tang-Li-1** [share121/inter-knot#1399] [其他]
![Screenshot_20240811_161854](https://github.com/user-attachments/assets/af1fda38-a0d4-4bb6-9394-c73744d798b9)
怎么一直加载啊

**Comment by gaojiaobei** [comment]
应该是没梯子导致的，绳网只有在人少的时候可能才能直接进

**Comment by ZMJ2333** [comment]
Steam++是好用的！
[这个网址](https://steampp.net/)
